GeneratorInterface/Core/plugins/ExternalGeneratorFilter.cc

```diff
@@ -287,13 +287,14 @@ void ExternalGeneratorFilter::streamEndRun(edm::StreamID iID, edm::Run const& iR
   }
 }
 void ExternalGeneratorFilter::globalEndRunProduce(edm::Run& iRun, edm::EventSetup const&) const {
-  iRun.emplace(runInfoToken_, std::move(runCache(iRun.index())->runInfo_));
+  iRun.emplace(runInfoToken_, runCache(iRun.index())->runInfo_);
```

Comment:
For anyone else who was wondering the removal of the `std::move`, the `GenRunInfoProduct` consists of 7 doubles, so moving brings no benefits over copying.